test/redeyed-smoke.js

```diff
@@ -53,6 +53,13 @@ test('esprima', function (t) {
 
   readdirp({ root: esprimadir, fileFilter: '*.js' })
     .on('data', function (entry) {
+      var invalidEspreeFiles = [
+          'lib/visitor-keys.js'
```

Comment:
Line 7: ` * The MIT license. Copyright (c) 2014 Ingvar Stepanyan` causes a false positive for the check `!(~result.indexOf('var '))` due to the name "Ing**var** Stepanyan"